services/QuillLMS/client/app/bundles/Lessons/components/classroomLessons/admin/show.tsx

```diff
@@ -62,6 +62,7 @@ class ShowClassroomLesson extends Component<any, any> {
     if (confirmation) {
       deleteLesson(this.props.match.params.classroomLessonID)
       window.location.href = `${window.location.origin}/lessons/#/admin/classroom-lessons/`
```

Comment:
This line was probably intended to accomplish the same thing, right? We should probably delete it if it's not doing anything.

Reply:
This line is still being used, it sets the new URL of the browser. We just need to add a hard reload with POST data so that's why I added the new line underneath.